src/OrchardCore.Modules/OrchardCore.Roles/Views/Items/SelectUsersInRoleTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,16 @@
+@using OrchardCore.Roles.Workflows.ViewModels
+@model SelectUsersInRoleTaskViewModel
+
+<div class="mb-3" asp-validation-class-for="OutputKeyName">
+    <label asp-for="OutputKeyName">@T["OutputKeyName"]</label>
+    <input type="text" asp-for="OutputKeyName" class="form-control code" />
+    <span asp-validation-for="OutputKeyName"></span>
+    <span class="hint">@T["The Output Key Name to save list. With Liquid support."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="Roles">
+    <label asp-for="Roles">@T["Roles"]</label>
+    @await Component.InvokeAsync("SelectRoles", new { selectedRoles = Model.Roles, htmlName = Html.NameFor(m => m.Roles), except = new[] { "Anonymous", "Authenticated" } })
+    <span asp-validation-for="Roles"></span>
+    <span class="hint">@T["The Roles to search for. With Liquid support."]</span>
```

Comment:
Maybe

Choose the roles used to identify users during list generation. With Liquid support.